skins/oasis/css/core/ads-porvata.scss

```diff
@@ -0,0 +1,282 @@
+@import '../../../../extensions/wikia/ArticleVideo/styles/video-shared-variables';
+
+$wikia-top-ads-index: 2;
+
+$default-progress-bar-color: #b7b7b7;
+$default-progress-bar-current-time-color: #666;
+$outstream-progress-bar-color: #666;
+$outstream-progress-bar-current-time-color: #b7b7b7;
+$video-player-button-size: 24px;
+
+$floating-ad-margin: 20px;
+$floating-ad-index: $wikia-top-ads-index + 10;
+$floating-ad-video-width: 320px;
+$floating-ad-animation-speed: $video-collapse-animation-speed;
+$floating-ad-featured-video-gap: $video-container-width;
+
+$vpaid-z-index: 10;
+$video-interactive-area-z-index: $vpaid-z-index + 1;
+
+@mixin absolute-fill() {
```

Comment:
what do you think about extracting it into a separate file that can be imported in other places?